package.json

```diff
@@ -5,7 +5,9 @@
   "homepage": "https://github.com/mieszko4/react-native-zoom-us",
   "main": "index.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepack": "npx tsc --noEmit false",
+    "postpublish": "npm run clean",
+    "clean": "npx module-kit-clean",
```

Comment:
there is no need for this `npx module-kit-clean` with `npmignore` and `.gitignore` defined

Reply:
this is not related to ignore files.
This is easy way to get rid of generated output. Very useful for development.

From other point, react-native has good bundler, which removes one common requirement.
So if  you don't publish locally, I can remove it, because it is not necessary to clean CI

Reply:
Why would I get rid of generated output if it is hidden for me in my editor and is not published to npm (as expected)?
So even if I publish locally these files will not be picked up because they are ignored in `npmignore`
There is really no need for that.

Reply:
I work a lot with tsconfig + babel, or just tsconfig node libs.
So for me it makes huge difference.

But for react-native, there no some bugs that I get from babel, so nevermind. I will remove.